llama_stack/providers/adapters/inference/ollama/ollama.py

```diff
@@ -31,11 +30,12 @@
 
 
 class OllamaInferenceAdapter(Inference, RoutableProviderForModels):
-    def __init__(self, url: str) -> None:
+    def __init__(self, url: str, preload_models: List[str] = None) -> None:
```

Comment:
We don’t want to be changing the signature of init methods since all other providers would have to do the same. Does it make sense to add an bootstrap call on a distribution which can then be hooked up to providers that may want to download models? Cc @ashwinb

Reply:
This can be taken care of by the `register_model()` call introduced in #201

Reply:
You're making a good point. I'll submit a PR later.